app/subscriber.py

```diff
@@ -58,7 +58,28 @@ def receipt_to_case(message: Message):
     log = log.bind(questionnaire_id=questionnaire_id, created=time_obj_created, tx_id=tx_id, case_id=case_id)
 
     metadata['response_datetime'] = time_obj_created
```

Comment:
There's no reason to modify the metadata dictionary here, it doesn't get used again.

Reply:
done